hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/IncrementalContainerReportHandler.java

```diff
@@ -80,11 +80,16 @@ public void onMessage(final IncrementalContainerReportFromDatanode report,
         processContainerReplica(dd, replicaProto, publisher);
       } catch (ContainerNotFoundException e) {
         success = false;
-        LOG.warn("Container {} not found!", replicaProto.getContainerID());
+        LOG.warn("Container {} not found!",
+            replicaProto.getContainerID());
       } catch (NodeNotFoundException ex) {
         success = false;
         LOG.error("Received ICR from unknown datanode {}",
             report.getDatanodeDetails(), ex);
+      } catch (ContainerReplicaNotFoundException e){
+        success = false;
+        LOG.warn("Container {} replica not found!",
+            replicaProto.getContainerID());
```

Comment:
Based on the CI logs generated by the previous commit (#5756e86), the CheckStyle tool only reported that this line didn't meet the character length limit (<=80). I think it is only necessary to adjust this part.
![image](https://user-images.githubusercontent.com/14295594/98562054-943f5b00-22e4-11eb-87fc-77c5743170a2.png)